includes/admin/class-wc-payments-admin.php

```diff
@@ -485,6 +485,7 @@ public function register_payments_scripts() {
 		}
 
 		$account_status_data = $this->account->get_account_status_data();
+		$account_status_data['deposits']['completed_waiting_period'] = false;
```

Comment:
Assuming this is dummy/test data?

Reply:
TY, Good catch! The dangers of using `git commit -a`!